File: python/src/tests/t_stop.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

atrshmlog.stop()

maxindex = atrshmlog.get_statistics_max_index()

stats = atrshmlog.get_statistics()

print('stop : ' + str(stats[11]) + ' : ')

print (' ')

exit(0)

# end of test

File: perl/src/tests/t_stop.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

Atrshmlog::stop();

$maxindex = Atrshmlog::get_statistics_max_index();

@stats = Atrshmlog::get_statistics();

print "stop : $stats[11] : \n";

print " \n";

exit (0);

# end of main
